room/views.py

```diff
@@ -69,7 +69,7 @@ def invite(request):
                 if not invitations:
                     # erm, kind of random but just want to get it working
                     code = ''.join(random.choice('abcdefghijklmnopqrstuvwxyz123456789') for i in range(9))
-                    invitation = Invitation.objects.create(room=room, email=email, code=code)
+                    invitation = Invitation.objects.create(room=room, email=email, invitation_code=code)
 
                     messages.info(request, 'Your new Groover at: ' + email +  ' was invited')
```

Comment:
Although pre-existing I would have thought an f-string would be an improvement:
```python
messages.info(request, f'Your new Groover at: {email} was invited')
```

Reply:
Yes, I need to break the habit of just concatenating them. I'll try and hunt down the others.